src/services/NumberService.ts

```diff
@@ -189,6 +221,22 @@ export class NumberService {
             });
     }
 
+    /**
+     * Retrieves the last plate serial number
+     */
+    public getLastPlateSerialNumber(): Promise<PlateSerialNumber> {
```

Comment:
we could probably also 'flatten this' using an async block.
```
public async getLastPlateSerialNumber(): Promise<PlateSerialNumber> {
        try {
            const data = await this.dbClient.get({testNumberKey: NUMBER_KEY.PLATE_SERIAL_NUMBER})
            if (!data.Item) return Configuration.getInstance().getPlateSerialNumberInitialValue();
            else return data.Item as PlateSerialNumber
        } catch(e) {
            throw this.formatAWSError(e);
        }
    }
```
This would work I think.